### PR TITLE
Add script to migrate v8 mappings with logicalReferences

### PR DESCRIPTION
**Pull request by simonvadee**
```
USAGE:\n> yarn run-ts scripts/migrateLogicalReferencesV8.ts <mapping.json> DEST
```

### CODE REVIEW

**Review by Jasopaum — COMMENTED** [review]
Have you tested the script on a mapping?
You can clean up the comment logs 🙂

**Comment by simonvadee** [comment]
removed the commented logs.
Yes I've tried it on a mapping and nico has tested it on osiris! I asked him to make sure references were binded by running an ETL, i'm still waiting for his feedback before merging this

**Comment by simonvadee** [comment]
@nriss has verified that reference binding is working, let's go!

**Review by Jasopaum — APPROVED** [review]

**Pull request merged**
